pulsar-client/src/main/java/org/apache/pulsar/client/impl/ClientCnx.java

```diff
@@ -1265,4 +1278,165 @@ private void checkRequestTimeout() {
     }
 
     private static final Logger log = LoggerFactory.getLogger(ClientCnx.class);
+
+    /**
+     * Check client connection is now free. This method may change the state to idle.
+     * This method will not change the state to idle.
```

Comment:
Shall we state more clear on whether his method WILL or WILL NOT change the state to idle?

Reply:
> Shall we state more clear on whether his method WILL or WILL NOT change the state to idle?

Already fixed, thanks.